### PR TITLE
fix case progress e2e tests

### PR DESCRIPTION
**Pull request by shravanmechineni**
https://tools.hmcts.net/jira/browse/DTSPB-2650

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,9 +43,9 @@
...
-    "test-e2e-caseProgressStandard": "E2E_TEST_PATH=./paths/caseprogress/caseProgressStandardPath.js E2E_TEST_TIME_TO_WAIT_FOR_TEXT=0 E2E_AUTO_DELAY_ENABLED=false BO_E2E_TEST_ACTION_WAIT_TIME=0 NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run-multiple parallel -c ./src/test/end-to-end/ --steps --reporter mochawesome",
-    "test-e2e-caseProgressReenter": "E2E_TEST_PATH=./paths/caseprogress/caseProgressReenterPath.js E2E_TEST_TIME_TO_WAIT_FOR_TEXT=0 E2E_AUTO_DELAY_ENABLED=false BO_E2E_TEST_ACTION_WAIT_TIME=0 NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run-multiple parallel -c ./src/test/end-to-end/ --steps --reporter mochawesome",
-    "test-e2e-caseProgressAppStopped": "E2E_TEST_PATH=./paths/caseprogress/caseProgressApplicationStoppedPath.js E2E_TEST_TIME_TO_WAIT_FOR_TEXT=0 E2E_AUTO_DELAY_ENABLED=false BO_E2E_TEST_ACTION_WAIT_TIME=0 NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run-multiple parallel -c ./src/test/end-to-end/ --steps --reporter mochawesome",
```
Comment (isha2040) [inline package.json:48]: why is -multiple parallel removed ?

Comment (shravanmechineni) [reply package.json:48]: I just removed it while testing it locally and there is no need for multiple parallel as it's just 1 test only (e.g. E2E_TEST_PATH=./paths/caseprogress/caseProgressStandardPath.js)
However, there is no effect on nightly builds as the tests are configured to run from bin/run-e2e-tests.sh script.

Comment (isha2040) [reply package.json:48]: ok I will take your lead here . Thanks

**Review by isha2040 — APPROVED** [review]

**Review by Mitesh1909 — COMMENTED** [review]
src/test/end-to-end/pages/utility/navigateToCase.js
How come you have removed most of this code? My understanding was that it was there to cater for the issues we had on successfully executing in the pipeline?

**Comment by shravanmechineni** [comment]
> src/test/end-to-end/pages/utility/navigateToCase.js How come you have removed most of this code? My understanding was that it was there to cater for the issues we had on successfully executing in the pipeline?

It seems like there is no option to search a specific case with id now, so I think it's no longer a need to use that code.

**Review by Mitesh1909 — APPROVED** [review]

**Pull request merged**